include/dump_offload.hpp

```diff
@@ -165,6 +165,7 @@ class Handler : public std::enable_shared_from_this<Handler>
 
     void cleanupSocketFiles()
     {
+        unixSocket.close();
```

Comment:
This should be done at line 181 ? when you successfully remove the unixSocketPath

Reply:
need to close socket first then remove socket file

Reply:
ok got it, moved unixSocket.close() inside file exists checks